src/pages/questionPage.js

```diff
@@ -23,13 +23,18 @@ export const initQuestionPage = (userInterface) => {
     .addEventListener('click', nextQuestion)
 };
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
   currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  isSelectedOneAnswer=true;
+}
 }
 const nextQuestion = () => {
```

Comment:
Now that I see your solution, are we checking somewhere that the user selected one answer before moving to the following question, or we don't care about that?